index.js

```diff
@@ -73,6 +73,6 @@ function fastifyGracefulShutdown(fastify, opts, next) {
 }
 
 module.exports = fp(fastifyGracefulShutdown, {
-  fastify: '^1.0.0 || ^2.0.0',
+  fastify: '^1.0.0 || ^2.0.0 || ^3.0.0',
```

Comment:
It's fine for me to release that change as breaking change. Please only add `^3.0.0`.

Reply:
Sure will ditch other changes then

Reply:
Because `fastify-plugin` 2.0.0 only supports fastify 3.0 see https://github.com/fastify/fastify-plugin

Reply:
Yes I was upgrading fastify in our project and it broke so headed over here 😅